src/server/src/main/java/io/cassandrareaper/resources/RepairScheduleResource.java

```diff
@@ -506,10 +526,13 @@ public Response deleteRepairSchedule(
         .build();
   }
 
-  private DateTime getNextActivationTime(Optional<String> scheduleTriggerTime) throws IllegalArgumentException {
+  private DateTime getNextActivationTime(
+      Optional<String> scheduleTriggerTime, Optional<String> clientTime)
+      throws IllegalArgumentException {
     DateTime nextActivation;
     if (scheduleTriggerTime.isPresent()) {
       nextActivation = DateTime.parse(scheduleTriggerTime.get());
+      nextActivation = adjustTimeDifferenceOnNextActivation(nextActivation, clientTime);
       LOG.info("first schedule activation will be: {}", RepairRunStatus.dateTimeToIso8601(nextActivation));
     } else {
       nextActivation = DateTime.now().plusDays(1).withTimeAtStartOfDay();
```

Comment:
i think this line deserves a comment, referencing that it's forced to UTC from https://github.com/thelastpickle/cassandra-reaper/blob/master/src/server/src/main/java/io/cassandrareaper/ReaperApplication.java#L142


Although that line in ReaperApplication really needs to be replaced with 
```
    TimeZone.setDefault(TimeZone.getTimeZone("UTC"));
    DateTimeZone.setDefault(DateTimeZone.UTC);```

Reply:
I would also look at improving that line of code as well
```
nextActivation = DateTime.now().plusDays(1).withTimeAtStartOfDay();
```

What we want is midnight local time, not midnight UTC. 
To make that possible we want need to save the original timezone in `ReaperApplication` before forcing it to UTC. The original saved timezone could then be made available via `AppContext`.